test/modeles/caracteristiquesComplementaires.spec.js

```diff
@@ -50,7 +50,7 @@ describe("L'ensemble des caractéristiques complémentaires", () => {
     describe("quand aucune entité externe n'a été saisie", () => {
       it('détermine le statut de saisie de manière standard', () => {
         const caracteristiques = new CaracteristiquesComplementaires({ hebergeur: 'Un hébergeur' });
-        expect(caracteristiques.statutSaisie()).to.equal(InformationsHomologation.A_COMPLETER);
+        expect(caracteristiques.statutSaisie()).to.not.equal(InformationsHomologation.COMPLETES);
```

Comment:
Je comprends qu'on ne veut pas avoir à retoucher les tests trop souvent… mais ça m'embête qu'on fragilise le test en vérifiant une inégalité plutôt qu'une égalité. Y a-t-il une raison pour laquelle on ne veut pas vérifier directement la valeur de `caracteristiques.statutSaisie()` ?